bokehjs/webpack.config.js

```diff
@@ -1,34 +0,0 @@
-require("./ts-node").register({project: "./gulp/tsconfig.json"});
-
-const paths = require("./gulp/paths");
-const {build_dir} = paths;
-
-const webpack = require("webpack");
-const {join} = require("path");
-
-const config = {
-  entry: {
-    "bokeh":         paths.lib.bokehjs.main,
-    "bokeh-api":     paths.lib.api.main,
-    "bokeh-widgets": paths.lib.widgets.main,
-    "bokeh-tables":  paths.lib.tables.main,
-    "bokeh-gl":      paths.lib.gl.main
-  },
-  output: {
-    filename: "[name].js",
-    path: build_dir.js,
-    library: "Bokeh",
-    libraryTarget: "umd"
-  },
-  resolve: {
-    modules: [build_dir.tree, "node_modules"]
-  },
-  plugins: [
-    new webpack.optimize.CommonsChunkPlugin({
-      name: "bokeh",
-    }),
-    new webpack.IgnorePlugin(/moment$/)
-  ]
-};
-
-module.exports = config;
```

Comment:
This could be moved to a wiki if we want to keep it, as long as this still works.

Reply:
I've added it here for now:

https://github.com/bokeh/bokeh/wiki/webpack.config.js

But we should find a way to support this in a more integrated way (i.e. make sure it works, and have a documentation section on it) since it seems to come up reasonably often.